website/docs/jellyfish/api/blockchain.md

```diff
@@ -268,6 +269,26 @@ interface MempoolTx {
 }
 ```
 
+## getMempoolInfo
+
+Returns details on the active state of the TX memory pool.
+
+```ts="client.blockchain.getMempoolInfo"
+interface blockchain {
+  getMempoolInfo(): Promise<MempoolInfo>
```

Comment:
Add a space between `getMempoolInfo` and `()`

Reply:
Added space in between the method name and the parenthesis.